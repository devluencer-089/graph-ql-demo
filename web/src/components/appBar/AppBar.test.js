import React from 'react';
import { createShallow } from 'material-ui/test-utils';
import AppBar from './AppBar';

describe('AppBar Component', () => {
  let shallow = createShallow({ dive: true });
  it('should match snapshot', () => {
    let eut = shallow(<AppBar />);

    expect(eut).toMatchSnapshot();
  });
});
