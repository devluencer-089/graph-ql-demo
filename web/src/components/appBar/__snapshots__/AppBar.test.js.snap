// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar Component should match snapshot 1`] = `
<div
  className="ResponsiveDrawer-root-1"
>
  <WithStyles(AppBar)
    className="ResponsiveDrawer-appBar-2"
  >
    <WithStyles(Toolbar)>
      <WithStyles(IconButton)
        aria-label="open drawer"
        className="ResponsiveDrawer-navIconHide-3"
        color="inherit"
        onClick={[Function]}
      >
        <pure(Menu) />
      </WithStyles(IconButton)>
      <WithStyles(Typography)
        color="inherit"
        noWrap={true}
        variant="title"
      >
        Senacor-In
      </WithStyles(Typography)>
    </WithStyles(Toolbar)>
  </WithStyles(AppBar)>
  <Hidden
    implementation="js"
    lgDown={false}
    lgUp={false}
    mdDown={false}
    mdUp={true}
    smDown={false}
    smUp={false}
    xlDown={false}
    xlUp={false}
    xsDown={false}
    xsUp={false}
  >
    <WithStyles(Drawer)
      ModalProps={
        Object {
          "keepMounted": true,
        }
      }
      anchor="left"
      classes={
        Object {
          "paper": "ResponsiveDrawer-drawerPaper-5",
        }
      }
      onClose={[Function]}
      open={false}
      variant="temporary"
    >
      <WithStyles(Component) />
    </WithStyles(Drawer)>
  </Hidden>
  <Hidden
    implementation="css"
    lgDown={false}
    lgUp={false}
    mdDown={false}
    mdUp={false}
    smDown={true}
    smUp={false}
    xlDown={false}
    xlUp={false}
    xsDown={false}
    xsUp={false}
  >
    <WithStyles(Drawer)
      classes={
        Object {
          "paper": "ResponsiveDrawer-drawerPaper-5",
        }
      }
      open={true}
      variant="permanent"
    >
      <WithStyles(Component) />
    </WithStyles(Drawer)>
  </Hidden>
  <main
    className="ResponsiveDrawer-content-6"
  >
    <div
      className="ResponsiveDrawer-toolbar-4"
    />
  </main>
</div>
`;
